plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -43,6 +44,8 @@ type PluginFactoryArgs struct {
 	NodeInfo                       predicates.NodeInfo
 	PVInfo                         predicates.PersistentVolumeInfo
 	PVCInfo                        predicates.PersistentVolumeClaimInfo
+	StorageClassInfo               predicates.StorageClassInfo
```

Comment:
I don't know why the "Info" interface is needed instead of just passing in the Lister.  It seems like it's just a direct passthrough to the Lister...  But it's what was done for PV, PVC.